ai_genomics/getters/data_getters.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pickle
+from fnmatch import fnmatch
+import boto3
+import json
+
+import pandas as pd
+
+from ai_genomics import bucket_name
+
+#####################################################################
+
+s3 = boto3.resource("s3")
+
+def get_s3_dir_files(s3, bucket_name, dir_name):
+    """
+    get a list of all files in bucket directory.
+    s3: S3 boto3 resource
+    bucket_name: The S3 bucket name
+    dir_name: bucket directory name
+    """
+    dir_files = []
+    my_bucket = s3.Bucket(bucket_name)
+    for object_summary in my_bucket.objects.filter(Prefix=dir_name):
+        dir_files.append(object_summary.key)
+
+    return dir_files
+
+def load_s3_data(bucket_name, file_name):
+    """
+    Load data from S3 location.
+    s3: S3 boto3 resource
+    bucket_name: The S3 bucket name
+    file_name: S3 key to load
+    """
+
+    obj = s3.Object(bucket_name, file_name)
+    if fnmatch(file_name, "*.csv"):
+        return pd.read_csv(os.path.join("s3://" + bucket_name, file_name))
+    elif fnmatch(file_name, "*.tsv.zip"):
+        return pd.read_csv(
+            os.path.join("s3://" + bucket_name, file_name), compression="zip", sep="\t"
+        )
```

Comment:
Minor formatting edit plus I don't think `os.path.join` is required here as the path formatting isn't platform dependent.

```suggestion
    if fnmatch(file_name, "*.csv"):
        return pd.read_csv(f"s3://{bucket_name}/{file_name}")
    elif fnmatch(file_name, "*.tsv.zip"):
        return pd.read_csv(
            f"s3://{bucket_name}/{file_name}",
            compression="zip",
            sep="\t",
        )
```